packages/unit-vitest/package.json

```
@@ -39,18 +39,18 @@
   },
   "devDependencies": {
     "@types/lodash-es": "^4.17.12",
-    "@vitest/ui": "^2.0.5",
-    "@vue/test-utils": "^2.4.4",
+    "@vitest/ui": "^3.0.8",
+    "@vue/test-utils": "^2.0.5",
     "quasar": "^2.14.6",
     "rimraf": "^5.0.5",
-    "vitest": "^2.0.5",
+    "vitest": "^2.4.6",
```

This should match the `@vitest/ui` version.
```suggestion
    "vitest": "^3.0.9",
```